README.md

```diff
@@ -750,35 +753,35 @@ in various programming languages.
   ([bibtex](https://dblp.uni-trier.de/rec/conf/fossacs/MatacheS19.html?view=bibtex))
   ([pdf](http://www.cs.ox.ac.uk/people/samuel.staton/papers/fossacs-2019.pdf))
 
-* **On the expressive power of user-defined effects: effect handlers, monadic reflection, delimited control** (JFP, ICFP 2017 special issue)  
+* **On the expressive power of user-defined effects: Effect handlers, monadic reflection, delimited control** (JFP, ICFP 2017 special issue)  
   by Yannick Forster, Ohad Kammar, Sam Lindley, and Matija Pretnar  
   ([doi](https://doi.org/10.1017/S0956796819000121))
   ([dblp](https://dblp.uni-trier.de/rec/journals/jfp/0002KLP19.html))
   ([bibtex](https://dblp.uni-trier.de/rec/journals/jfp/0002KLP19.html?view=bibtex))
   ([pdf](http://homepages.inf.ed.ac.uk/slindley/papers/effmondel-jfp.pdf))
 
-* **Dijkstra Monads for All** (ICFP 2019)  
-  by Kenji Maillard, Danel Ahman, Robert Atkey, Guido Martínez, Cătălin Hriţcu, Exequiel Rivas and Éric Tanter  
+* **Dijkstra monads for all** (ICFP 2019)  
+  by Kenji Maillard, Danel Ahman, Robert Atkey, Guido Martínez, Cătălin Hriţcu, Exequiel Rivas, and Éric Tanter  
   ([doi](https://doi.org/10.1145/3341708))
   ([dblp](https://dblp.uni-trier.de/rec/journals/pacmpl/MaillardAAMHRT19.html))
   ([bibtex](https://dblp.uni-trier.de/rec/journals/pacmpl/MaillardAAMHRT19.html?view=bibtex))
   ([pdf](https://arxiv.org/abs/1903.01237))
```

Comment:
If we choose to be consistent wrt. arxiv/pdf then this should say `[arxiv]` as well.

Reply:
(there may be lingering instance of this inconsistency floating around the document)

Reply:
I would suggest that `arxiv` points to the main arXiv page, and that we provide direct links for `doi` and `pdf` as a fallback. I.e:
- if there is an arXiv submission, `arxiv` should point to `https://arxiv.org/abs/xxxx.yyyy` where a PDF is just a click away,
- if the paper is published somewhere, `doi` should point there, otherwise it should point to the one provided by arXiv: `https://doi.org/aaa.bbb/arXiv.xxxx.yyyy`
- if the published PDF is freely available somewhere (ideally by publisher, or on author's homepage), `pdf` should point there. Otherwise, if it is on arXiv, it should point to `https://arxiv.org/pdf/xxxx.yyyy.pdf`.

WDYT?

Reply:
Sounds reasonable to me. Then this entry should read `[arxiv]`, whilst the other three above are OK as is, right?

Reply:
Yes, except that I've added an `[arxiv]` link with `/abs/` as well.